tools/matchms/matchms_filtering_wrapper.py

```diff
@@ -84,6 +89,11 @@ def main(argv):
         if args.reduce_to_top_n_peaks:
             spectrum = reduce_to_number_of_peaks(spectrum_in=spectrum, n_max=args.n_max)
 
+        if args.derive_precursor_mz_from_parent_mass:
+            spectrum.set("parent_mass", int(float(spectrum.get('parent_mass'))))
```

Comment:
This line has to be removed to keep the feature functional.

Reply:
removed conversion to int in 9e579c4